tests/support/server.rs

```diff
@@ -54,11 +55,34 @@ pub fn spawn(txns: Vec<Txn>) -> Server {
                 thread::park_timeout(dur);
             }
 
+            // the server will encode the body as chunks if:
+            //   - a `write_timeout` has been supplied
+            //   - a `chunk_size` has been supplied
+            // 
+            // the server won't read headers so if the response doesn't specify `Transfer-Encoding`
+            // then it'll misbehave and send an invalid response.
```

Comment:
Good idea :+1: I can search the response bytes for a `Transfer-Encoding` header and assert it's present.